dev-docs/modules/floors.md

```diff
@@ -1182,6 +1182,56 @@ The module will do this by leveraging the already-existing implementation for an
 The PrebidServerBidAdapter calls `getFloor()` like any other bid adapter
 and passes it to the server side as imp.bidfloor and imp.bidfloorcur.
 
+### Publisher Interface
+
+Obviously if a publisher is defining their own floors, then all of the fields in the floors schema are part of the publisher interface.
+
+Even if a publisher is using a floors provider, they may wish to provide additional data:
+1. default floor data if dynamic data fails to load on time
+2. global floorMin: allows the publisher to constrain dynamic floors with a global min
+3. impression-level floor min (PBJS 6.24+): allows the publisher to constrain dynamic floors with an adunit-specific value
+
+Here's an example covering the first two scenarios:
+```
+pbjs.setConfig({
+      floors: {
+          enforcement: {
+             floorDeals: false //default to false
+          },
+          floorMin: 0.05,      // global default
+	  auctionDelay: 100,   // in milliseconds
+          endpoint: {          // where to get the dynamic floors
+            url: 'https://floorprovider.com/a1001-mysite.json'
+          },
+          data: {
+	     ... default floors data here ...
```

Comment:
As there is an example config for everything in the config, maybe add an example here as well 😀